src/route-handlers/describe-domain/describe-domain.ts

```diff
@@ -19,9 +20,14 @@ export async function describeDomain(
 
     return NextResponse.json(res.domain);
   } catch (e) {
+    logger.error(
+      { requestParams: decodedParams, error: e },
+      'Error fetching domain info'
+    );
+
     return NextResponse.json(
       {
-        error:
+        message:
           e instanceof GRPCError ? e.message : 'Error fetching domain info',
         cause: e,
```

Comment:
maybe cause can renamed to errors array and reused for all types of errors

Reply:
The ErrorOptions supports a "cause" field by default so I think it makes more sense to call it "cause" at the response level too